apps/mobile/src/libs/components/button/styles.ts

```diff
@@ -3,13 +3,29 @@ import { StyleSheet } from "react-native";
 import { BaseColor } from "~/libs/enums/enums";
 
 const styles = StyleSheet.create({
-	btn: {
-		alignItems: "center",
-		backgroundColor: BaseColor.BLACK,
-		padding: 10,
+	bgWhite: {
+		backgroundColor: BaseColor.BG_WHITE,
 	},
-	label: {
-		color: "white",
+	colorBlack: {
+		color: BaseColor.BLACK,
+	},
+	colorWhite: {
+		color: BaseColor.BG_WHITE,
+	},
+	p1: {
+		padding: 1,
+	},
```

Comment:
We already have p2 in globalStyles. Can we use it instead?

Reply:
If it is allowed to have a slight difference from padding size in design, I'd better use globalStyles.